src/po/pages/calculator.page.js

```diff
@@ -0,0 +1,93 @@
+    get frankfurt () { return $('//*[@id="select_container_132"]/md-select-menu/md-content/md-optgroup/md-option[26]') }; 
+    get committedUsage () { return $('//md-select[@placeholder="Committed usage"]') };
+    get oneyear () { return $('//div[@class="md-select-menu-container md-active md-clickable"]//md-option[2]') };
+    get addToEstimateBtn () { return $('button.md-raised.md-primary.cpc-button.md-button.md-ink-ripple') };
+
+    get checkRegion () { return $('//*[@id="compute"]/md-list/md-list-item[1]/div[1]') };
+    get checkCommitmentTerm () { return $('//*[@id="compute"]/md-list/md-list-item[3]/div[1]') };
+    get checkVmClass () { return $('//*[@id="compute"]/md-list/md-list-item[4]/div[1]') };
+    get checkInstanceType () { return $('//*[@id="compute"]/md-list/md-list-item[5]/div[1]') };
+    get checkLocalSsd () { return $('//*[@id="compute"]/md-list/md-list-item[7]/div[1]') };
+    get checkRentalAmount () { return $('div.cpc-cart-total > h2 > b') };
+
+    get btnEmailEstimate () { return $('//button[@id="Email Estimate"]') };
+    get inputEmail () { return $('//form[@name="emailForm"]/md-content/div[3]/md-input-container/input') };
+    get btnSendEmail () { return $('form > md-dialog-actions > button:last-child') };
+
+    async switchFrames(){
+        let iframeOne = await this.iFrame1;
+        await browser.switchToFrame(iframeOne);
+        let iframeTwo = await this.iFrame2;
+        await browser.switchToFrame(iframeTwo);
+    };
+
+    async goOutFrames(){
+        await browser.switchToFrame(null);
+    };
+
+    async addEstimate (num, text, location) {
+        await this.computeEngine.click();
+        await this.numberOfInstances.click();
+        await this.numberOfInstances.setValue(num);
+        await this.instanceFor.click();
+        await this.instanceFor.setValue(text);
+        await this.series.click();
+        await this.n1Series.click();
+        await this.instanceType.click();
+        await this.n1standard8.waitForClickable( { timeout: 3000 } );
+        await this.n1standard8.click();
+        await this.addGPUs.click();
+        await this.gpuType.click();
+        await this.nvidiaTesla.waitForClickable( {timeout: 6000} );
+        await this.nvidiaTesla.click();
+        await this.numberGPU.click();
+        await this.oneNumber.click();
+        await this.localSSD.click();
+        await this.twoGB.waitForClickable( {timeout: 6000} );
+        await this.twoGB.click();
+        await this.datacenterLocation.click();
+        await this.inputLocation.click();
+        await this.inputLocation.setValue(location);
+        await this.frankfurt.click();
+        await this.committedUsage.click();
+        await this.oneyear.click();
+        await this.addToEstimateBtn.click();
+    };
+
+    async sendEmail(email){
+        await this.btnEmailEstimate.click();
+        await browser.pause(2000);
```

Comment:
Please replace all pause methods this with some waiters (waitFor.. or waitUntil).